docs/conf.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+#
+# Hypothesis Client documentation build configuration file, created by
+# sphinx-quickstart on Thu Mar  9 18:01:59 2017.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+import os
+# import sys
+
+from recommonmark.parser import CommonMarkParser
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# sys.path.insert(0, os.path.abspath('.'))
+
+# on_rtd is whether the docs are being built on readthedocs.org (or whether
+# someone is building them locally), this line of code grabbed from
+# docs.readthedocs.org
+on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = ['sphinx.ext.todo']
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+source_parsers = {
+    '.md': CommonMarkParser,  # Add Markdown support to Sphinx.
+}
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+# source_suffix = ['.rst', '.md']
+source_suffix = ['.rst', '.md']
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = u'Hypothesis Client'
+copyright = u'2017, Hypothesis Project Contributors'
+author = u'Hypothesis Project Contributors'
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
+version = u'1.2.0'
+# The full version, including alpha/beta/rc tags.
+release = u'1.2.0'
```

Comment:
The autogeneration tool requires a version num to be given (not sure if it's actually needed, maybe not, maybe it is) but it's not actually used anywhere (as with h, on rtd it'll show as just one version called latest). It may be possible to pull it from somewhere - this is a Python file that can contain arbitrary code - we'd have to make sure that ran on readthedocs too, but it should be doable. (I'd say don't bother though - we aren't using this number.)